File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.py
class Solution(object):
    def findNumbers(self, nums):
        n = len(nums)
        count = 0
        for num in nums:
            digit = 0
            while(num > 0):
                num /= 10
                digit+=1
            if(digit % 2 == 0):
                count+=1
        return count

        

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int n = nums.length;
        int count = 0;
        int digits = 0;
        for(int i = 0; i<n; i++){
            digits = String.valueOf(nums[i]).length();
            if(digits % 2 == 0){
                count++;
            }
        }
        return count;
    }
}